processors/src/main/resources/DeltaBuilderBody.vm

```diff
@@ -61,23 +61,27 @@
     #end
     public ${return_type} set$alias(final $field.type newValue) {
         // first check if we already have a delta of this field
-        ${field.boxedType} curValue;
```

Comment:
Yes because I won't really be using them outside.  For example, the generated code now looks like this:

```
        if (deltaMap().containsKey("dateTime")) {
            java.util.Date curValue = (java.util.Date) deltaMap().get("dateTime").getOldValue();
            Delta.OP op = compare(curValue, newValue);
            if (op != Delta.OP.NOOP) {
                addDelta(new Delta(op, "dateTime", curValue, newValue));
            } else {
                removeDelta("dateTime");
            }
        } else {
            java.util.Date curValue = from().dateTime;
            Delta.OP op = compare(curValue, newValue);
            if (op != Delta.OP.NOOP) {
                addDelta(new Delta(op, "dateTime", curValue, newValue));
            }
        }
```